src/css/chat-panel.css

```
@@ -0,0 +1,1146 @@
+  background: var(--code-background, var(--pre-background));
+  color: var(--code-font-color);
+  padding: 1em;
+  border-radius: 0.5rem;
+  overflow-x: auto;
+  font-size: 0.85em;
+  line-height: 1.6;
+  margin: 0.75rem 0;
+  font-family: var(--monospace-font-family, var(--font-mono, "JetBrains Mono", monospace));
+  font-weight: lighter;
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.11.1)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)